### ISSUE TITLE
Concurrent log entries to the cache made by the same user

### ISSUE THREAD

**Issue by rapotek** [opencaching/opencaching-pl#1600]
When the "Found it" log is submitted by a user, the next log attempt for the same cache made by the user has "Found It" and "Didn't find it" not available - which is correct. But consider the scenario:

1. Open two 'Post new log' pages for the cache in separate window/card for not found cache - there are both options available.
2. Choose "Found it" in the first window and submit.
3. Few minutes later choose "Didn't find it" in the second window and submit.

The result is: the cache is indicated as found by the user but the last submitted log is "Didn't find it".

The similar effect can be of course achieved by submitting sequentially logs "Didn't find it" and "Found it", when the latter date is set by the user to earlier than the former one, but at least `date_created` field is correct in this case.

In short: the previously saved log types and dates should be checked on the server side on submit.

**Comment by kojoty** [comment]
@rapotek - please explain it to me: what should be exactly checked on server side?

**Comment by deg-pl** [comment]
@rapotek @kojoty there is one more issue - see e-mail from me.

**Comment by rapotek** [comment]
@kojoty - in short it should be checked if:

- there already exists the entry which prevents submitting the given type of log for user,
- (optionally) the chronology of log entries will be consistent after saving submitted entry
